packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -1102,10 +1102,11 @@ abstract class ResidentRunner {
       'flutter',
       'png',
     );
-    final List<FlutterView> views = await device
-      .vmService.getFlutterViews();
+    List<FlutterView> views = <FlutterView>[];
```

Comment:
can we document this some way to avoid future me "optimizing" this code back to the previous state?

Reply:
done! hopefully the test will help too